scripts/ci/pylint_todo.txt

```diff
@@ -20,7 +20,7 @@
 ./airflow/contrib/hooks/opsgenie_alert_hook.py
 ./airflow/contrib/hooks/pinot_hook.py
 ./airflow/contrib/hooks/qubole_check_hook.py
-./airflow/contrib/hooks/redshift_hook.py
+./airflow/providers/amazon/aws/hooks/redshift.py
```

Comment:
@ratb3rt -> maybe you can fix pylint errors alongside the move? That would be nice one to do.

Reply:
I think that one PR should introduce only one change.  otherwise, it will not be possible to find the cause of the problem if it were entered during the correction of errors. I sometimes intentionally compare the contents of a file to be sure that the change describes what it does.

Reply:
As mik-laj suggested - l'll leave the linting to a separate batch of PRs, targeting already-migrated modules.